crates/hcl-edit/src/parser/expr.rs

```diff
@@ -334,7 +334,7 @@ fn binary_operator(input: &mut Input) -> ModalResult<BinaryOperator> {
         '+' => empty.value(BinaryOperator::Plus),
         '-' => empty.value(BinaryOperator::Minus),
         '*' => empty.value(BinaryOperator::Mul),
-        '/' => empty.value(BinaryOperator::Div),
+        '/' => not(alt(('/', '*'))).value(BinaryOperator::Div),
```

Comment:
You can replace the `alt` with `one_of`, which is more efficient and also reads nicer IMHO.

```suggestion
        '/' => not(one_of(b"/*")).value(BinaryOperator::Div),
```